infra/archiveStorage.bicep

```
@@ -0,0 +1,129 @@
+param location string = resourceGroup().location
+
+var tags = {
+  'cost-category': 'core'
+}
+
+@allowed([
+  'Premium_LRS'
+  'Premium_ZRS'
+  'Standard_GRS'
+  'Standard_GZRS'
+  'Standard_LRS'
+  'Standard_RAGRS'
+  'Standard_RAGZRS'
+  'Standard_ZRS'
+])
+param skuName string
+
+var unique = substring(uniqueString(resourceGroup().id), 0, 8)
+
+resource blobStorage 'Microsoft.Storage/storageAccounts@2022-09-01' = {
+  name: 'blobsswwebsite${unique}'
```

```suggestion
  name: 'stsswwebsite${unique}'
```
Change prefix for storage accounts - https://learn.microsoft.com/en-us/azure/cloud-adoption-framework/ready/azure-best-practices/resource-abbreviations

---

Done 👍